ext/src/zpages/README.md

```diff
@@ -1,5 +1,5 @@
 # zPages
-> Last updated 6/26/20
+> Last updated 7/13/20
```

Comment:
Do we need this or we can rely on `git blame`?

Reply:
Good idea, let's do `git blame`